src/main/scala/no/ndla/validation/TagRules.scala

```diff
@@ -72,6 +72,8 @@ object TagAttributes extends Enumeration {
   val DataAuthors = Value("data-authors")
   val DataArticleId = Value("data-article-id")
   val DataPath = Value("data-path")
+  val DataFormat = Value("data-code-format")
```

Comment:
Format her er feks programmeringsspråket for kodeblokker og text for inline kode?

Reply:
ja. Format er det i drop-down menyen for valg av programmeringsspråk og content er selve innholdet i blokken. 

Siden det format i den editoren og her: https://github.com/NDLANO/frontend-packages/blob/master/packages/ndla-editor/src/CodeblockEditor/languages.tsx

Reply:
burde kanskje kalt det DataCodeFormat?

Reply:
Har ingen praktisk betydning.